msg/tools/uorb_rtps_message_ids.json

```diff
@@ -0,0 +1,352 @@
+{
+  "rtps": [
+    {
+      "msg": "actuator_armed",
+      "id": 1
```

Comment:
Could we do this as a metadata field in the message itself? Alternatively something in the CMakeLists.txt might also make sense. https://github.com/PX4/Firmware/blob/master/msg/CMakeLists.txt#L35-L140
I'm worried about maintaining the list over time.